src/main/resources/db/migration/V42_7__Create_post_reaction_table.sql

```diff
@@ -0,0 +1,6 @@
+create table if not exists post_reaction
+(
+    id UUID PRIMARY KEY,
+    id_post UUID REFERENCES post(id),
+    type VARCHAR(255)
```

Comment:
L13 - Should be an enum, not a varchar

take the example of hei-admin
https://github.com/hei-school/hei-admin-api/blob/preprod/src/main/resources/db/migration/V0_3__Create_fee_table.sql
check also the implementation of the attribute with type Enum in Class Fee in the model